soc/nxp/imxrt/imxrt11xx/soc.c

```diff
@@ -398,6 +398,11 @@ static ALWAYS_INLINE void clock_init(void)
 	rootCfg.mux = kCLOCK_LPI2C5_ClockRoot_MuxOscRc48MDiv2;
 	rootCfg.div = 1;
 	CLOCK_SetRootClock(kCLOCK_Root_Lpi2c5, &rootCfg);
+
+	/* Configure Lpi2c6 using Osc24M */
```

Comment:
This really should be in a separate PR from this one- it is clearly something we are missing on the RT1170 clock setup, but this PR is just enabling the camera driver